templates/definition/meter/homematic.yaml

```
@@ -44,3 +53,6 @@ render: |
   {{ if ne .password "" }}
   password: {{ .password }}
   {{ end }}
+  {{- if ne .cache "" }}
```

```suggestion
  {{- if .cache }}
```

Wird ohnehin immer gezogen da Du oben schon den Default 1s setzt. Oder auf `ne "1s"` prüfen